src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -10,21 +10,77 @@ import {
   isMultiSendTxInfo,
   isMultiSigExecutionDetails,
 } from 'src/logic/safe/store/models/types/gateway.d'
-import { InlineEthHashInfo } from './styled'
+import { InlineEthHashInfo, StyledGridRow } from './styled'
 import { NOT_AVAILABLE } from './utils'
 import TxShareButton from './TxShareButton'
 import { IS_PRODUCTION } from 'src/utils/constants'
 import TxInfoMultiSend from './TxInfoMultiSend'
 import DelegateCallWarning from './DelegateCallWarning'
+import styled from 'styled-components'
+import { getByteLength } from 'src/components/DecodeTxs' // must pass to utils
+import { generateSignaturesFromTxConfirmations } from 'src/logic/safe/safeTxSigner'
+import { makeConfirmation } from 'src/logic/safe/store/models/confirmation'
 
 type Props = { txDetails: ExpandedTxDetails }
 
+const FlexWrapper = styled.div<{ margin: number }>`
```

Comment:
Can you add a comment if this reacts to the class name-based styles within the transaction list? I think we should definitely rely purely on `styled-components` so it's nice to see this.

Reply:
Can you explain in more detail the first part of the comment?

Reply:
This was just a comment. I like that you used `styled-components` here 😃